modules/reindex/src/test/java/org/elasticsearch/index/reindex/AsyncBulkByScrollActionTests.java

```diff
@@ -171,7 +171,6 @@ private void setupClient(ThreadPool threadPool) {
     @After
     public void tearDownAndVerifyCommonStuff() throws Exception {
         client.close();
```

Comment:
a client was created with the same `threadPool` that is terminated a line below.
a client terminates a threadPool as well in NoOpClient.close() so this is possibly not needed

It might as well stay, but I think it is unecessary